manifests/namespace-install/workflow-controller-rbac/workflow-controller-role.yaml

```diff
@@ -28,6 +28,7 @@ rules:
       - ""
     resources:
       - configmaps
+      - namespaces
```

Comment:
I don't think we should be doing this for namespaced installs, it feels like they wouldn't use this feature.

Reply:
Yes they won't use this feature, but they might still expect the dynamic limits to work? I think that is a fair expectation.

Reply:
Addressed